mm/kfence.c

```diff
@@ -119,8 +136,8 @@ static void __meminit kfence_unprotect(unsigned long addr)
 
 	addr_end = addr + PAGE_SIZE;
 	pte = lookup_address(addr, &level);
-	BUG_ON(!pte);
-	BUG_ON(level != PG_LEVEL_4K);
+	kfence_assert(pte);
```

Comment:
There should be some error recovery.

Reply:
With the suggestion for KFENCE_WARN, it'd be 'if (KFENCE_WARN(!pte)) { bail out; }'

Reply:
Done.